src/components/toggle.vue

```diff
@@ -19,14 +18,22 @@
 </template>
 
 <script lang="ts">
-import { Component, Prop, Vue } from 'vue-property-decorator';
+import { Component, Prop, Vue, Watch } from 'vue-property-decorator';
 
 @Component
 export default class Toggle extends Vue {
 
   @Prop({default: false, type: Boolean})
   value!: boolean;
 
+  @Watch('value')
+  on_value_changed(new_value: boolean, old_value: boolean) {
+    this.is_on = new_value;
+  }
+
+  @Prop({default: false, type: Boolean})
+  wee!: boolean;
```

Comment:
I don't think this variable is used anywhere in the toggle component.

Reply:
Correct, this branch actually isn't going to ever be merged because the changes were originally made on branch #15 (and I removed that property there as well).